### PR TITLE
[Release] 7/28

### PR DESCRIPTION
**Pull request by sukhada**

### CODE REVIEW

**Comment by cjmabry** [comment]
@sukhada was this ready for review or were you working on figuring out merge conflicts?

File: afteraction-share.html [hunk 1]
```diff
@@ -369,7 +369,9 @@ <h2>{{ page.custom_fields.share_slide_title }}</h2>
 			var winLeft = (screen.width / 2) - (popupWidth / 2);
 			window.open( destination , 'new-window-popup', 'top=' + winTop + ',left=' + winLeft + ',toolbar=0,status=0,resizable=yes,scrollbars=yes,width=' + popupWidth + ',height=' + popupHeight );
 			// send Google Analytics event
-			ga('send', 'event', 'scrolling-signup', 'share-facebook', window.location.pathname);
+			if (typeof ga !== 'undefined') {
```
Comment (cjmabry) [inline afteraction-share.html:372]: Might want to add below commentary about handling adblockers here, and might want to expand on what this is going for future devs.

What is this doing? Basically if an adblocker is present and blocks GA, this code block won't run? How is that different that what happens without this if block?

**Review by cjmabry — APPROVED** [review]
Looks good, a few clarifying questions but nothing blocking.

File: thanks.html [hunk 2]
```diff
@@ -264,8 +264,12 @@ <h4 class="icon-after icon-faded" data-icon="&#xe93F;">{% filter ak_text:"genera
 						<h4 class="icon-after icon-faded" data-icon="&#xE941;">{% filter ak_text:"general_tweet" %}Share on Twitter{% endfilter %}</h4>
 						</div>
 						<div class="box box-big bg-white border">
-							<textarea class="bg-ltgray border strong p" readonly>{{ page.followup.twitter_message }}</textarea>
-							<p><a class="button-share-medium button-share-twitter bg-twitter-icon bg-twitter-color no-margin-bottom"href="/share/link?type=tw&amp;page_name={{page.name}}&amp;action_id={{action.id}}&amp;akid={{akid}}">{% filter ak_text:"general_tweet" %}Tweet{% endfilter %}</a></p>
+							{% if page.followup and page.followup.twitter_message %}
+								<textarea class="bg-ltgray border strong p" readonly>{{ page.followup.twitter_message }}</textarea>
+							{% else %}
+								<textarea class="bg-ltgray border strong p" readonly>{{ page.followup.share_title_value }} {{ page.canonical_url }}</textarea>
```
[the remark this answers is not in the document]
Comment (sukhada) [reply thanks.html:270]: It seems like it is based on: https://github.com/350org/ak_intl_v3/pull/366/files/fdd992a8508e1b7613730cf41a2ad395393a3715#diff-e24ea3aee006d6a0ff673819ee97abc88406ae07f5c34d2ccce80840515f805cL252

File: afteraction-share.html (hunk 1 above)
Comment (sukhada) [reply afteraction-share.html:372]: If it runs without checking that `ga` is `undefined`, there's an exception that prevents the rest of the JS from executing correctly. I can add a comment explaining that!

File: privacy.html [hunk 3]
```diff
@@ -114,3 +114,9 @@
 
 {% endwith %}
 {% endfilter %}
+
+<style>
```
[the remark this answers is not in the document]
Comment (sukhada) [reply privacy.html:118]: I can add a comment in a future PR so we can avoid having to open another PR against develop and merging to develop again. Can add a comment for this as well as the GA undefined check

**Comment by rachelhbd** [comment]
@stbensonimoh let us know when you've reviewed, thanks much!

File: afteraction-share.html (hunk 1 above)
Comment (stbensonimoh) [reply afteraction-share.html:372]: ++ to adding the comment @sukhada. It will be very helpful.

**Review by stbensonimoh — APPROVED** [review]
It looks good to me!
I agree with @sukhada that we should avoid opening a new PR against `dev` only to merge into `dev` again. But definitely support adding comments like @cjmabry suggested.

**Pull request merged**